README.md

```diff
@@ -64,3 +64,61 @@ ExHuobi.Margin.Rest.Order.bulk_create(
 # Cancel multiple orders
 ExHuobi.Margin.Rest.Order.bulk_cancel(%{"order-ids": [70664141188, 70664141185]})
 ```
+
+## Futures Exchange API
+```elixir
+config = %ExHuobi.Config{
+  api_key: System.get_env("HUOBI_API_KEY"),
+  api_secret: System.get_env("HUOBI_API_SECRET")
+}
+# Get account position
+symbol = "BTC|ETH..."
+ExHuobi.Futures.Rest.Account.get_position(symbol, config)
+
+# Create new order
+order = %{
+        "contract_type" => "this_week",
+        "direction" => "Buy",
+        "lever_rate" => 5,
+        "offset" => "open",
+        "order_price_type" => "limit",
+        "price" => 5000,
+        "symbol" => "BTC",
+        "volume" => 1
+      }
+ExHuobi.Futures.Rest.Order.create_order(order, config)
+
+# Create multiple orders
+orders = [
+      %{
+        "contract_type" => "this_week",
+        "direction" => "Buy",
+        "lever_rate" => 1,
+        "offset" => "open",
+        "order_price_type" => "limit",
+        "price" => 5000,
+        "symbol" => "BTC",
+        "volume" => 1
+      },
+      %{
+        "contract_type" => "next_week",
+        "direction" => "Buy",
+        "lever_rate" => 5,
+        "offset" => "open",
+        "order_price_type" => "limit",
+        "price" => 5000,
+        "symbol" => "BTC",
+        "volume" => 1
+      },
+    ]
+
+
+ExHuobi.Futures.Rest.Order.create_bulk_orders(orders, config)
+
+# Cancel order
+
+order_id = 1234567890
+
+ExHuobi.Futures.Rest.Order.create_bulk_orders(order_id, config)
```

Comment:
@lamtrhieu one last small comment. This is order bulk creation, why passing in only 1 single order id ?

Reply:
@kimquy ah, it's a typo. Thanks for spot this out. Let me fix it.